### PR TITLE
NO-TICKET: fixes broken localstack

### PR DESCRIPTION
**Pull request by sakibstark11**
Fixes #
Localstack was broken for awhile, this pr fixes it

Resolves #
Used when the changes in the pull request indirectly resolve another issue.

- [x] Does not depend on other PRs
- [ ] Depends on other PR being merged first: #

## What kind of change does this PR introduce (check at least one)

- [x] Bugfix
- [x] Improvement
- [ ] New feature
- [ ] Refactor
- [ ] Infrastructure
- [ ] Other (please specify)

## How would you describe the changes in a changelog (1-2 lines)?

Provide a brief overview of the changes introduced in this pull request.

## Related Issues/PRs (optional)

Link to the relevant issue(s) or provide a clear description of the problem or feature being addressed.

## How to test

- [ ] Needs regression testing (add QA needed label)

## Checklist

- [x] I have tested these changes locally and they function as expected.
- [ ] I have added or updated necessary documentation to reflect the changes.
- [ ] I have followed the project's coding style and guidelines.
- [ ] I have added appropriate test coverage to ensure the changes are functioning correctly.
- [ ] I have checked for any potential conflicts with other branches or pull requests.
- [ ] I have updated necessary OpenAPI specs

## Reviewer Request (optional)

If you have any specific requests or suggestions for reviewers, mention them here.

## Screenshots (optional)

If applicable, provide any additional screenshots or GIFs that showcase the changes made in this pull request.

## Demo Link (optional)

If applicable, provide a link to a live demo or deployment where the changes can be reviewed.

## Known Issues (optional)

If there are any known issues or limitations with the changes made in this pull request, describe them here.

## Additional Notes (optional)

Add any additional notes or comments you think are relevant to this pull request.

## Deploy risk & follow up

Please assess risk & follow up on deployment of this PR. Elaborate your choice if selecting anything other than 'No risk'.

- [x] No risk
- [ ] Low
- [ ] High

### CODE REVIEW

**Review by mahinuzzaman — DISMISSED** [review]

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -32,3 +32,4 @@ jobs:
     secrets: inherit
     with:
       runner_image_name: ${{ needs.build-push-docker-image.outputs.runner_image_name }}
+      deploy_localstack: true
```
Comment (mahinuzzaman) [inline .github/workflows/ci.yml:35]: Adding localstack deploy check for every CI can slow down the development process. Can we add localstack deploy check for stage and PR only?

Comment (sakibstark11) [reply .github/workflows/ci.yml:35]: yeah please, not sure how to do that though

Comment (mahinuzzaman) [reply .github/workflows/ci.yml:35]: Ok vai, I'll update it

File: .tool-versions
Comment (melon-ruet) [inline .tool-versions:1]: We do not need it

**Review by melon-ruet — CHANGES_REQUESTED** [review]

File: .tool-versions
Comment (sakibstark11) [reply .tool-versions:1]: I do, i use it for formatting.

**Pull request merged**